src/supertux/menu/options_menu.cpp

```diff
@@ -179,6 +183,77 @@ OptionsMenu::OptionsMenu(bool complete) :
     resolutions.push_back(fullscreen_size_str);
   }
 
+  // Sound Volume
+  sound_volumes.clear();
```

Comment:
Maybe the colorchannel item could be used for that, well, it wouldn't be draggable and the values would go from 0 to 1 instead of 0% to 100%. I personally wouldn't mind choosing between fixed values, but I don't see why music_volumes should be different than sound_volumes. And when it's like this, the int values can be precalculated, so we don't need to extract them from the string on each change.